litentry/core/ruleset-build/src/discord.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020-2022 Litentry Technologies GmbH.
```

Comment:
I'm thinking to have a more clear structure where each ruleset has its own file, like:
```
preset/r1.rs
preset/r2.rs
```
or 
```
template/r1.rs
template/r2.rs
```

We can have a `common/` or `util/` mod if something is in common

I think that would be better than discord.rs => we have/will have more discord-related rulesets

Reply:
This can be refactored later when new/other ruleset added in.